src/Files.Shared/Services/IJumpListService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Files.Shared.Services;
+
+public interface IJumpListService
+{
+	Task AddFolderAsync(string path);
+	Task RefreshPinnedFoldersAsync();
+	Task RemoveFolderAsync(string path);
+	Task<IEnumerable<string>> GetFoldersAsync();
```

Comment:
Use `IAsyncEnumerable<>` here, and `CancellationToken` in other places as well

Reply:
Because a try-catch is necessary in the implemented method body, I will keep it as is for now. `yield return` can't be used.